vmdb/spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -0,0 +1,2760 @@
+require "spec_helper"
+
+describe ApplicationHelper do
+  before do
+    controller.send(:extend, ApplicationHelper)
+    self.class.send(:include, ApplicationHelper)
+  end
+
+  def method_missing(sym, *args)
```

Comment:
Can you explain what this is for?  I'm concerned it will affect other tests (I'm not sure the scoping of this method_missing)